testscript/build.gradle

```diff
@@ -16,8 +16,16 @@ dependencies {
 }
 
 compileKotlin {
-    kotlinOptions.jvmTarget = "1.8"
+    kotlinOptions {
+        jvmTarget = "1.8"
+        apiVersion = "1.4"
+        languageVersion = "1.4"
+    }
```

Comment:
Not sure if this should be more global--but basically every Kotlin sub-project has to explicitly set this and this change is consistent with existing package structure.

Reply:
Once the planner refactor is closed, I will begin the build refactor. These options should be in the buildSrc like was done with the PIG refactor https://github.com/partiql/partiql-ir-generator/blob/main/buildSrc/src/main/kotlin/pig.conventions.gradle.kts#L26-L31

Reply:
I guess that whole file shows how to fix the issue in one place and apply to all gradle subprojects via a conventions plugin